Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/result/AggregatedRiskResult.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.risk.result
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+
+// TODO("Adjust Types")
+data class AggregatedRiskResult(
+    val totalRiskLevel: RiskCalculationParametersOuterClass.NormalizedTimeToRiskLevelMapping.RiskLevel,
+    val totalMinimumDistinctEncountersWithLowRisk: Int,
+    val totalMinimumDistinctEncountersWithHighRisk: Int,
+    val mostRecentDateWithLowRisk: Int?,
```

Comment:
Shouldn't this be some kind of Date object rather than integer?

Reply:
true. changed it to instant and adjusted corresponding use.